modules/codepipeline/templates/buildspec_scan.yml

```diff
@@ -16,4 +16,4 @@ phases:
       - export PATH=$PATH:./sonar-scanner-4.7.0.2747-linux/bin/
   build:
     commands:
-      - sonar-scanner -Dsonar.projectKey=revature-sample-lambda -Dsonar.sources=. -Dsonar.login=${SONARQUBE_TOKEN} -Dsonar.organization=${SONAR_ORG} -Dsonar.host.url=${SONARQUBE_ENDPOINT}
\ No newline at end of file
+      - sonar-scanner -Dsonar.projectKey=revature-sample-lambda -Dsonar.sources=. -Dsonar.login=${SONARQUBE_TOKEN} -Dsonar.organization=${SONAR_ORG} -Dsonar.host.url=${SONARQUBE_ENDPOINT} -Dsonar.qualitygate.wait=true
```

Comment:
This property is the configuration that tells the `sonar-scanner` tool to wait for the qualitygate results before continuing. If the quality gate is not passed, then the pipeline will fail.